src/DrupalVersionChanger.php

```diff
@@ -488,23 +496,19 @@ private function replaceWildcardVersionsInComposerJson(): void
         // Extract locked versions from composer.lock.
         $locked_versions = $this->extractLockedVersions();
         // Update wildcard versions for all other required packages in composer.json.
-        foreach ($json_data['require'] as $package => $version) {
-            if ($this->hasWildcardVersion($version) && isset($locked_versions[$package])) {
+        foreach (['require', 'require-dev'] as $require_type) {
+          // Update wildcard versions for all other required-dev packages in composer.json.
+          if (isset($json_data[$require_type])) {
+            foreach ($json_data[$require_type] as $package => $version) {
+              if ($this->hasWildcardVersion($version) && isset($locked_versions[$package])) {
                 $exact_wersion = $locked_versions[$package];
```

Comment:
```suggestion
                $exact_version = $locked_versions[$package];
```

This isn't an issue introduced by your changes, but I noticed a minor typo in the code: the variable should be `..version` instead of `..wersion`.

Thanks.